compiler/rustc_mir_transform/src/reveal_all.rs

```diff
@@ -36,6 +36,6 @@ impl<'tcx> MutVisitor<'tcx> for RevealAllVisitor<'tcx> {
 
     #[inline]
     fn visit_ty(&mut self, ty: &mut Ty<'tcx>, _: TyContext) {
-        *ty = self.tcx.normalize_erasing_regions(self.param_env, ty);
+        *ty = self.tcx.try_normalize_erasing_regions(self.param_env, *ty).unwrap_or(ty);
```

Comment:
Can you add a comment explaining that the MIR we are visiting might have impossible-to-satiafy `where` clauses?